pom.xml

```diff
@@ -304,15 +305,9 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
```

Comment:
This BouncyCastle dependency needs to remain. It is used in for testing the crypto code that allows switching to a different JCE provider. The JUnit tests is created to skip the test if there's no BC library present, but we generally want those tests to run. 
Also, IIRC, this was the most recent version that work when we were running against Java 7. It's possible that a newer version may work with Java 8, but given that this is only used for testing, using an older version is not a big deal.

Reply:
Restored